### DISCUSSION TITLE
Latin1 characters not being rendered in Quarto

### DISCUSSION THREAD

**Discussion by igorlaltuf** [quarto-dev/quarto-cli#4759] [Q&A] [answered]
I'm trying to generate a pdf report in Portuguese, but words with accents are not rendered. 

Is there any way to change the encoding used in rendering to Latin1?

![image](https://user-images.githubusercontent.com/22546557/224444686-772a5601-9a4f-4bd7-b726-2b1340566ecf.png)

**Comment by dragonstyle** [comment]
I think the issue is likely the font (you need to use a unicode font in your PDF document). If you're using the default out of the box PDF generation in Quarto, you'll be using the `xelatex` PDF engine and you can set the fonts like so:

````markdown
---
title: Test unicode content
format:
  pdf:  
    mainfont: DejaVuSerif.ttf
    sansfont: DejaVuSans.ttf
    monofont: DejaVuSansMono.ttf
---

## probabilístico

probabilístico 

```
probabilístico
```
````

HTH!

**Comment by igorlaltuf** [comment]
@dragonstyle  Apparently that's the problem, but the error persists even with the suggested font. Do you know where I can find the unicode fonts available in Quarto?
![image](https://user-images.githubusercontent.com/22546557/224451689-416d7fc5-5925-4a85-9099-059f8cebf872.png)

**Comment by igorlaltuf** [comment, marked as answer]
I was able to make it work by reopening the file in RStudio with UTF-8 encoding (even using it in Windows).
